projects/plugins/jetpack/extensions/blocks/conversation/participants.json

```diff
@@ -3,15 +3,18 @@
 	"list": [
 		{
 			"slug": "participant-0",
-			"label": "Speaker 1"
+			"label": "Speaker 1",
+			"value": "<strong>Speaker 1</strong>"
```

Comment:
yeah, sounds good to me. I left the plain participant (label) because maybe it could be useful to print a participant list in areas without formatting. In case we don't wanna that, we can remove it. 👍